src/document/advanced.rs

```diff
@@ -9,37 +13,39 @@ use crate::{
 };
 use itertools::EitherOrBoth;
 
+/// IronOxide Advanced Document Operations
+///
+/// # Key Terms
+/// - ID     - The ID representing a document. It must be unique within the document's segment and will **not** be encrypted.
+/// - Name   - The human-readable name of a document. It does not need to be unique and will **not** be encrypted.
+/// - EDEKs  - Encrypted document encryption keys used by unmanaged document encryption and decryption.
```

Comment:
EDEKS are produced by unmanged document encryption and used by unmanaged document decryption

Reply:
Fixed